test/extensions.jl

```diff
@@ -0,0 +1,32 @@
+using NEOs
+using DataFrames
+using Test
+
+using NEOs: read_radec_mpc, RadecMPC, read_radar_jpl, RadarJPL
+
+@testset "DataFramesExt" begin
+
+    # RadecMPC
+    radec_2023DW = read_radec_mpc(joinpath("data", "RADEC_2023_DW.dat"))
+
+    df_2023DW = DataFrame(radec_2023DW)
+
+    @test nrow(df_2023DW) == length(radec_2023DW)
+    @test all(names(df_2023DW) .== String.(fieldnames(RadecMPC)))
+
+    recovered_2023DW = RadecMPC(df_2023DW)
+
+    @test recovered_2023DW == radec_2023DW
```

Comment:
I would suggest to add a very simple test of selecting observations (one test for optical and one for radar) by observatory and checking that these kind of operations return what we expect. For example, let's say that we have 10 observations from observatory A on a given night, then we should be able to get these observations via a select on the DataFrame.

Reply:
I've added two simple tests using `Query.jl` to filter observations by date and observatory.